tools/codegen/core/gen_grpc_tls_credentials_options.py

```diff
@@ -247,7 +247,7 @@ class DataMember:
     ),
     DataMember(
         name="identity_certificate_provider",
```

Comment:
I suggest moving these two new fields to be right below the existing `certificate_provider` field, since they are all the same type and for the same purpose.